contents/select.md

```diff
@@ -0,0 +1,191 @@
+## Select {#sec:select}
+
+Whereas `filter` removes rows, `select` removes columns.
+However, `select` is much more versatile than just removing columns, as we will discuss in this section.
+First, let's create a dataset with multiple columns:
+
+```jl
+@sco responses()
+```
+
+Here, the data represents answers on five questions (`q1`, `q2`, ..., `q5`) in a questionnaire.
+We will start by "selecting" a few columns from this dataset.
+Again, we use symbols to specify columns:
+
+```jl
+s = "select(responses(), :id, :q1)"
+sco(s, process=without_caption_label)
+```
+
+We can also use strings if we want:
+
+```jl
+s = """select(responses(), "id", "q1", "q2")"""
+sco(s, process=without_caption_label)
+```
+
+To select everything _except_ one or more columns, use `Not` with either a single column name:
```

Comment:
The `Not` `Between` etc are called column selectors in DataFrames.jl. If you type `?select` in julia with the `using DataFrames` you get:

```julia
standard column selectors (integers, Symbols, strings, vectors of integers, vectors of Symbols, vectors of
       strings, All, Cols, :, Between, Not and regular expressions)
```

Maybe we should introduce this term to the reader?

Reply:
#54